WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
That is an exaggeration, other religions have incest too, and using incest creates separate opinion modifiers.